src/routes/index.tsx

```diff
@@ -26,8 +27,10 @@ const routes = (
 const AppRoutes = () => {
   return (
     <BrowserRouter>
-      <ScrollTop />
-      <Routes>{routes}</Routes>
+      <VaultInfosProvider>
```

Comment:
Move this context to file `src/modules/vault/routes.tsx`

Reply:
Done